Yoast/wordpress-seo
# Update the Dashboard Widget with new design.

## Summary

This PR can be summarized in the following changelog entry:

* Updated the Dashboard Widget with a new design.

## Relevant technical choices:

* Uses a React components from `Yoast/yoast-components`.
* Does not use the background image as there's too much variation in dashboard column widths to reliably display it.
* Keeps the Ryte assessment.
* Fetches the latest yoast.com blog posts by RSS which requires CORS-headers. In case of no CORS headers it fails gracefully by not displaying the blog part.

## Test instructions

This PR can be tested by following these steps:

* Build all assets ( `grunt build` ).
* Go to the dashboard.
* You should see the new dashboard widget featuring a stacked progress bar.

Fixes #7549
